hw/ip/prim/util/sparse-fsm-encode.py

```diff
@@ -219,6 +240,29 @@ def main():
 );
 '''.format(state_str))
 
+    elif args.language == "c":
+        print("// Encoding generated with\n"
+            "// $ ./sparse-fsm-encode.py -d {} -m {} -n {} -s {} --language=c\n"
```

Comment:
just noting that this line can get really long, depending on the seed (and that could cause lint issues).
so we may have to split this onto two lines